### ISSUE TITLE
PyGLM doesn't compile with python 3.9.1

### ISSUE THREAD

**Issue by decaruju** [Zuzu-Typ/PyGLM#110]
pip install PyGLM returns an error during compilation.

The error log is very long, I can paste it if you need it.

**Comment by Zuzu-Typ** [comment]
Hm.. It might be because the version it's trying to install doesn't support Python 3.9 yet (on some operating systems).

Could you please try again with `pip install PyGLM==2.0.0a3`?

If that doesn't work, then I would like to ask you to post the error message and give me a little info on your OS and compiler setup.

Thanks in advance!

Cheers,
--Zuzu_Typ--

**Comment by decaruju** [comment]
I can't seem to install the pre-release version, I get 
```
pip install pyglm==2.0.0a3 --pre
ERROR: Could not find a version that satisfies the requirement pyglm==2.0.0a3 (from versions: 0.1.0a1, 0.1.2a1, 0.2.0a1, 0.2.1a1, 0.3.0a1, 0.3.1a1, 0.3.2a1, 0.3.3a1, 0.3.4a1, 0.4b1, 0.4.1b1, 0.4.2b1, 0.4.3b1, 0.4.4b1, 0.4.5b1, 0.4.6b1, 0.4.7b1, 0.4.8b1, 0.5.0b1, 0.5.1b1, 0.5.2b1, 0.5.3b1, 0.6.0b1, 0.6.1b1, 0.7.1b1, 0.7.2b1, 1.0.0, 1.0.1, 1.1.0, 1.1.1, 1.1.2, 1.1.4, 1.1.5, 1.1.6, 1.1.7)
ERROR: No matching distribution found for pyglm==2.0.0a3
```
However I trust that the new version will fix the problem when it will be released, for now I'll keep using 3.8 when I need PyGLM.

Thanks!
